lib/r10k/source/branchname.rb

```diff
@@ -0,0 +1,112 @@
+require 'r10k/git'
+require 'r10k/environment'
+require 'r10k/util/purgeable'
+require 'r10k/util/core_ext/hash_ext'
+
+# This class controls the behaviour of branch names
+#
+# A branch name becomes the name of the environment on the file system.
+# The default branch name is the name of the source branch.
+# The name can be modified with a prefix, and by default will be automatically
+# modified if the branch name contains characters invalid for a file system.
+#
+class R10K::Source::BranchName
+
+  # @!attribute [r] name
+  #   @return [String] The raw name of the branch directly from the source
+  attr_reader :name
+
+  # @!attribute [r] remote
+  #   @return [String] The url of the svn remote. Only used to storage and
+  #     retrieval when working with this class.
+  attr_reader :remote
```

Comment:
This variable looks like it's coming from the `SVN::BranchName` class, but that class never used that value. We can omit it here.